audio_service/lib/audio_service.dart

```diff
@@ -587,6 +587,19 @@ class MediaItem {
   ///  * Network - http:// https:// etc.
   ///  * Android content URIs - content://
   ///
+  /// ## Speeding up Android content URI loading
+  ///
+  /// For Android content:// URIs, the plugin by default uses
+  /// `ContentResolver.openFileDescriptor`, which takes the direct URI
+  /// of an image.
+  ///
+  /// On Android >= 30 there's a `ContentResolver.loadThumbnail` function
+  /// which takes a URI of some content (for example, a song from `MediaStore`),
+  /// and returns a thumbnail for it.
+  ///
+  /// It is a noticeably faster to use this function. You can do enable this by putting
+  /// a `loadThumbnailUri` key into the [extras]. If `loadThumbnail` is not available,
+  /// it will just fallback to using `openFileDescriptor`.
```

Comment:
Maybe this would be more intuitive to make it a boolean, like `loadThumbnailUri: true`. If `false` or the key is absent, then it defaults to the old way.

Reply:
Sorry I meant still make it a key in extras, so key in quotes: `'loadThumbnailUri': true`.

Reply:
Currently, it can fallback to the `artUri` in case if `loadThumbnail` can't be used (due to API level, as marked in the comment above). If we make it a flag, this becomes a responsibility of the API user to detect this, which would be worse, it seems to me.

Reply:
Ah, I completely misunderstood the comment. I thought it said to just put the key into `extras` and hence the value was irrelevant. I see now that there is meant to be a URI as the value.